Makefile

```diff
@@ -88,6 +88,21 @@ pre_commit_nodeps: ## Run pre-commit tests without installing dependencies
 		pre-commit run --all-files; \
 	fi
 
+.PHONY: check_zuul_files
```

Comment:
I love the idea - maybe we should nudge that in a script though, to make things a bit more readable?

Reply:
That's right. Change done.